lib/controllers/journal.js

```diff
@@ -139,37 +141,43 @@ function notesMoodsSideEffects (req, res, next) {
   });
 }
 
-journal.get("/notes", auth.authorize("read"), notesMoodsSideEffects);
-journal.get("/moods", auth.authorize("read"), notesMoodsSideEffects);
-journal.get("/side-effects", auth.authorize("read"), notesMoodsSideEffects);
+journal.get("/notes", guard(["admin", "programAdministrator", "clinician"]), auth.authorize("read"), notesMoodsSideEffects);
+journal.get("/moods", guard(["admin", "programAdministrator", "clinician", "user"]), auth.authorize("read"), notesMoodsSideEffects);
+journal.get("/side-effects", guard(["admin", "programAdministrator", "clinician", "user"]), auth.authorize("read"), notesMoodsSideEffects);
 
 // Create a new journal entry for the specified patient (requires write access)
-journal.post("/",
+journal.post("/", guard(["admin", "programAdministrator", "clinician", "user"]),
     auth.authorize("write"), filterInput, authorizeFromData("write"), setRole,
 function (req, res, next) {
+    if (req.authClaims.scopes.every((scope) => scope === "user") && req.data && req.data.text) {
```

Comment:
NOTE: The important part of this logic being the check for `req.data.text` that differentiates this from a guard call without the scope `user`